test/parallel/test-http-malformed-request.js

```diff
@@ -50,7 +50,4 @@ server.on('listening', function() {
     c.end();
```

Comment:
You could also put `common.mustCall` here around the `function` within `c.on('connect')`.